jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -1329,12 +1325,15 @@ public void testGetTaskEvents() {
     	assertNotNull(workItemId);
 
     	UserTaskInstanceDesc userTask = runtimeDataService.getTaskByWorkItemId(workItemId);
-    	assertNotNull(userTask);
+        assertTask(userTask, processInstanceId);
 
     	List<TaskEvent> auditTasks = runtimeDataService.getTaskEvents(userTask.getTaskId(), new QueryFilter());
     	assertNotNull(auditTasks);
-        assertEquals(2, auditTasks.size());
-    	assertEquals(TaskEvent.TaskEventType.ADDED, auditTasks.get(0).getType());
+    	assertEquals(2, auditTasks.size());
+    	TaskEvent taskEvent = auditTasks.get(0);
+    	assertEquals(TaskEvent.TaskEventType.ADDED, taskEvent.getType());
+    	assertNotNull(taskEvent.getCorrelationKey());
+    	assertNotNull(taskEvent.getProcessType());
```

Comment:
Maybe assert the process type as well.

Reply:
processType here can be either PROCESS or CASE, so what is tested is that TaskEvent is filled as expected with a not null value

Reply:
But since you started process in this test you can expect PROCESS here, so the test would have even better coverage.

Reply:
you are right, changed